Analysis/src/EmbeddedBuiltinDefinitions.cpp

```diff
@@ -145,59 +145,59 @@ declare function loadstring<A...>(src: string, chunkname: string?): (((A...) ->
 declare function newproxy(mt: boolean?): any
 
 declare coroutine: {
-    create: <A..., R...>((A...) -> R...) -> thread,
-    resume: <A..., R...>(thread, A...) -> (boolean, R...),
+    create: <A..., R...>(f: (A...) -> R...) -> thread,
+    resume: <A..., R...>(co: thread, A...) -> (boolean, R...),
     running: () -> thread,
-    status: (thread) -> "dead" | "running" | "normal" | "suspended",
+    status: (co: thread) -> "dead" | "running" | "normal" | "suspended",
     -- FIXME: This technically returns a function, but we can't represent this yet.
-    wrap: <A..., R...>((A...) -> R...) -> any,
+    wrap: <A..., R...>(f: (A...) -> R...) -> any,
     yield: <A..., R...>(A...) -> R...,
     isyieldable: () -> boolean,
-    close: (thread) -> (boolean, any)
+    close: (co: thread) -> (boolean, any)
 }
 
 declare table: {
-    concat: <V>({V}, string?, number?, number?) -> string,
-    insert: (<V>({V}, V) -> ()) & (<V>({V}, number, V) -> ()),
-    maxn: <V>({V}) -> number,
-    remove: <V>({V}, number?) -> V?,
-    sort: <V>({V}, ((V, V) -> boolean)?) -> (),
-    create: <V>(number, V?) -> {V},
-    find: <V>({V}, V, number?) -> number?,
-
-    unpack: <V>({V}, number?, number?) -> ...V,
+    concat: <V>(t: {V}, sep: string?, f: number?, t: number?) -> string,
+    insert: (<V>(t: {V}, value: V) -> ()) & (<V>(t: {V}, i: number, v: V) -> ()),
+    maxn: <V>(t: {V}) -> number,
+    remove: <V>(t: {V}, i: number?) -> V?,
+    sort: <V>(t: {V}, f: ((V, V) -> boolean)?) -> (),
+    create: <V>(n: number, v: V?) -> {V},
+    find: <V>(t: {V}, v: V, i: number?) -> number?,
+
+    unpack: <V>(a: {V}, f: number?, t: number?) -> ...V,
     pack: <V>(...V) -> { n: number, [number]: V },
 
-    getn: <V>({V}) -> number,
-    foreach: <K, V>({[K]: V}, (K, V) -> ()) -> (),
-    foreachi: <V>({V}, (number, V) -> ()) -> (),
+    getn: <V>(t: {V}) -> number,
+    foreach: <K, V>(t: {[K]: V}, f: (key: K, value: V) -> ()) -> (),
+    foreachi: <V>(t: {V}, f: (index: number, value: V) -> ()) -> (),
```

Comment:
observation: these type definitions themselves actually deviate from the documentation - https://luau-lang.org/library#table-library